README.md

```diff
@@ -1 +1,92 @@
-# angular-resource-readme
\ No newline at end of file
+# $resource
+
+## Overview
+
+We've had a look at the awesome `$http` service that Angular provides, but there is also another awesome service under the name of `$resource`. Let's take a look at that!
+
+## Objectives
+
+- Describe $resource
+- Use $resource methods
+- Fetch data from an API
+- Update data from an API
+
+## What is $resource?
+
+$resource is a factory that creates a resource object for us to communicate with APIs. We pass through a URL and it gives us an object back that looks like this:
```

Comment:
At this point, we haven't defined what a factory is. Can this be added to a previous README?